ush/python/ocean/storm_tempZ70m.py

```diff
@@ -121,8 +120,8 @@
    plt.suptitle(storm.upper()+tcid.upper()+'  '+'Ver Hr '+"%03d"%(fhr)+'  (IC='+cycle+'): '+var_name+ ' & Change '+units,fontsize=15)
 
    ax121 = plt.subplot(121)
-   #(var1[k]*dumb).plot.contourf(levels=np.arange(0,850,50),cmap='RdBu_r')
-   kw = dict(levels=np.arange(np.floor(np.nanmin(var)),np.round(np.nanmax(var),1)+delta_var,delta_var))
+   kw = dict(levels=np.arange(10,25,2))
```

Comment:
Same as previous comment, the range of 70m temperatures from HSK's scripts is 15C to 29C. Should change this to be consistent.